zgw/object-management/src/main/kotlin/com/ritense/objectmanagement/service/ObjectManagementFacade.kt

```diff
@@ -0,0 +1,203 @@
+package com.ritense.objectmanagement.service
+
+import com.fasterxml.jackson.databind.JsonNode
+import com.ritense.objectenapi.ObjectenApiPlugin
+import com.ritense.objectenapi.client.ObjectRecord
+import com.ritense.objectenapi.client.ObjectRequest
+import com.ritense.objectenapi.client.ObjectWrapper
+import com.ritense.objectenapi.client.ObjectsList
+import com.ritense.objectmanagement.domain.ObjectManagement
+import com.ritense.objectmanagement.repository.ObjectManagementRepository
+import com.ritense.objecttypenapi.ObjecttypenApiPlugin
+import com.ritense.plugin.service.PluginService
+import mu.KotlinLogging
+import org.springframework.data.domain.PageRequest
+import org.springframework.stereotype.Service
+import org.springframework.web.reactive.function.client.WebClientResponseException
+import java.net.URI
+import java.time.LocalDate
+import java.util.UUID
+
+@Service
+class ObjectManagementFacade(
+    private val objectManagementRepository: ObjectManagementRepository,
+    private val pluginService: PluginService
+) {
+    val logger = KotlinLogging.logger {}
+
+    fun getObjectByUuid(objectName: String, uuid: UUID): ObjectWrapper {
+        val accessObject = getAccessObject(objectName)
+
+        return findObjectByUuid(accessObject = accessObject, uuid = uuid)
+    }
+
+    fun getObjectsByUuids(objectName: String, uuids: List<UUID>): ObjectsList {
+        val accessObject = getAccessObject(objectName)
+        val objects = mutableListOf<ObjectWrapper>()
+
+        uuids.forEach(){
+            objects.add(findObjectByUuid(accessObject = accessObject, uuid = it))
+        }
+
+        return ObjectsList(count = objects.size, results = objects)
+    }
+
+    private fun findObjectByUuid(accessObject: ObjectManagementAccessObject, uuid: UUID): ObjectWrapper {
+        val objectUrl = URI.create("${accessObject.objectenApiPlugin.url}objects/$uuid")
```

Comment:
Potential bug: This will fail if the objecten-api-plugin has been configured with an url that doesn't end with a `/`.

Maybe something like this is better:
```
UriComponentsBuilder
    .fromUri(objectenApiPlugin.url)
    .pathSegment("objects")
    .pathSegment(uuid.toString())
    .build()
    .toUri()
```

Reply:
Your suggestion has been implemented.